content/en/blog/preact-x.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Preact X, a story of stability
+date: 2024-05-24
+authors:
+  - Jovi De Croock
+---
+
+# Preact X, a story of stability
+
+A lot of you have been waiting for [Preact 11](https://github.com/preactjs/preact/issues/2621), announced in an issue opened
+way back in July 2020, and to be clear I was one of the most excited people for v11.
+When we started thinking about Preact 11 we believed that there was no way to introduce the changes we had in mind
+in Preact X without breaking changes, some of the things we had in mind:
+
+- Using a backing VNode structure to reduce GC, by doing this we'd only use the result of `h()` to update our backing-node.
+- Reconciler performance, adding fast paths for mounting/...
+- Some changes like removing `px` suffixing, `forwardRef` and breaking IE11 support.
+- Addressing event/child diffing bugs.
+
+The above were our goals, when breaking these down we can clearly see that there's a distinction here, only point 3 would
+be a breaking change for our consumers while point 1 would have a huge impact to folks building extensions like Preact Signals,
+Preact devtools, ... This could break a lot of the cool stuff our community is working on to extend Preact.
+
+As we were [experimenting](https://github.com/preactjs/preact/tree/v11) we went a new type of diffing, named
+[skew based diffing](https://github.com/preactjs/preact/pull/3388), we saw real performance
+improvements as well as it fixing a bunch of long-running bugs. As time went on and we invested more time in
+these experiments for Preact 11, we started noticing that the improvements we were landing didn't need to be exclusive to Preact 11.
+
+## Releases
+
+Since the aforementioned Preact 11 issue there have been 18 (!!) minor versions of Preact X.
+Many of them have been directly inspired by work done on Preact 11. Let's go over a few and look at the impact.
+
+### 10.5.0
+
+The introduction of [resumed hydration](https://github.com/preactjs/preact/pull/2754) this functionality basically allows suspending during
```

Comment:
```suggestion
The introduction of [resumed hydration](https://github.com/preactjs/preact/pull/2754) -- this functionality basically allows suspending during
```

Semicolon would work too I think, but needs something to break up the flow of the sentence.